defi/src/governance/icp.ts

```diff
@@ -1,82 +1,87 @@
 import axios from 'axios'
 import { GovCache, Proposal } from './types';
-// import ic from 'ic0';
+import ic from 'ic0';
```

Comment:
this brings additional dependency that I want to avoid, if you dont want to show slightly outdated total supply info, we can skip it completely? I dont think that metric is used much

Reply:
I found a workaround solution that fetches the up to date token balance but does not require an additional library